conf/deploy.json

```diff
@@ -6,6 +6,16 @@
             "data": {
                 "bucket": "content-api-dist"
             }
+        },
+        "content-api-dashboard-ami-update" : {
+            "type": "ami-cloudformation-parameter",
+            "data": {
+                "amiTags" : ["ubuntu-wily-java-8"],
```

Comment:
Sorry, `Recipe` is correct. Thanks to https://github.com/guardian/amigo/pull/33, we should also add `"AmigoStage": "PROD"` so we don't accidentally use an image that somebody built on their dev machine.